docs/en_US/NAS/Overview.rst

```diff
@@ -1,44 +1,59 @@
+
+However, it is pretty hard to use existing NAS work to help develop common DNN models. Therefore, we designed `Retiarii <https://www.usenix.org/system/files/osdi20-zhang_quanlu.pdf>`__, a novel NAS/HPO framework, and implemented it in NNI. It helps users easily construct a model space (or search space, tuning space), and utilize existing NAS algorithms. The framework also facilitates NAS innovation and is used to design new NAS algorithms.
+
 Overview
 --------
 
-Automatic neural architecture search is taking an increasingly important role in finding better models. Recent research has proved the feasibility of automatic NAS and has lead to models that beat many manually designed and tuned models. Some representative works are `NASNet <https://arxiv.org/abs/1707.07012>`__\ , `ENAS <https://arxiv.org/abs/1802.03268>`__\ , `DARTS <https://arxiv.org/abs/1806.09055>`__\ , `Network Morphism <https://arxiv.org/abs/1806.10282>`__\ , and `Evolution <https://arxiv.org/abs/1703.01041>`__. Further, new innovations keep emerging.
+There are three key characteristics of the Retiarii framework:
 
-However, it takes a great effort to implement NAS algorithms, and it's hard to reuse the code base of existing algorithms for new ones. To facilitate NAS innovations (e.g., the design and implementation of new NAS models, the comparison of different NAS models side-by-side, etc.), an easy-to-use and flexible programming interface is crucial.
+* Users can easily express model space on their original PyTorch/TensorFlow model. The model space could include both neural architectures and hyper-parameters.
+* Users can directly use many SOTA NAS algorithms to explore their model space.
+* There are system level optimizations in Retiarii to speed up the exploration process.
 
-With this motivation, our ambition is to provide a unified architecture in NNI, accelerate innovations on NAS, and apply state-of-the-art algorithms to real-world problems faster.
+There are two types of model space exploration approach: multi-trial NAS and one-shot NAS. Users can construct a model space and use either exploration appraoch to explore the space. Please refer to `Quick Start <./QuickStart.rst>`__ for how to run a NAS experiment on NNI.
 
-With the unified interface, there are two different modes for architecture search. `One <#supported-one-shot-nas-algorithms>`__ is the so-called one-shot NAS, where a super-net is built based on a search space and one-shot training is used to generate a good-performing child model. `The other <#supported-classic-nas-algorithms>`__ is the traditional search-based approach, where each child model within the search space runs as an independent trial. We call it classic NAS.
+.. Note:: `The previous NAS framework <https://nni.readthedocs.io/en/v1.9/nas.html>`__ is deprecated, we highly recommend users to move to this Retiarii framework.
```

Comment:
let's move this up front.

maybe change the tune to： NNI's future NAS supports will all based on Retiarii Framework, users who are still on early version using NNI NAS v1.0 shall migrate your work to Retiarii as soon as possible.

Reply:
updated